sandbox/src/routes/retrieveWorklist.js

```diff
@@ -7,7 +7,7 @@ module.exports = [
    */
   {
     method: 'POST',
-    path: '/STU3/ReferralRequest/$ers.fetchworklist',
+    path: '/FHIR/STU3/ReferralRequest/$ers.fetchworklist',
     handler: (request, h) => {
 
       const allowedBusinessFunctions = ["SERVICE_PROVIDER_CLINICIAN", "SERVICE_PROVIDER_CLINICIAN_ADMIN", "SERVICE_PROVIDER_ADMIN"]
```

Comment:
We'd also need to be careful around the retrieve ReferralRequest worklist endpoint